decentralized-message-queue/src/DMQ/Protocol/LocalMsgNotification/Client.hs

```diff
@@ -0,0 +1,58 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE PolyKinds #-}
+
+-- | todo haddock
+module DMQ.Protocol.LocalMsgNotification.Client
+  ( LocalMsgNotificationClient (..)
+  , LocalMsgNotificationClientStIdle (..)
+  , localMsgNotificationClientPeer
+  ) where
+
+import Data.List.NonEmpty (NonEmpty)
+
+import DMQ.Protocol.LocalMsgNotification.Type
+import Network.TypedProtocol.Peer.Client
+
+-- | todo haddock
+newtype LocalMsgNotificationClient m msg a = LocalMsgNotificationClient {
+  runMsgNotificationClient :: m (LocalMsgNotificationClientStIdle m msg a)
+  }
+
+
+-- | todo haddock
```

Comment:
Let's remove the `todo haddock`, they're not helpful when browsing documentation.